ADADAISY

```diff
@@ -0,0 +1,11 @@
+{
+    "project": "ADADAISY",
+    "tags": [
+        "ADADAISY",
+        "ADA DAISY"
```

Comment:
Please add a comma

Reply:
Hello! Added the comma, hopefully all is well! Thanks for your help!